glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -368,6 +380,30 @@ open class GleanInternalAPI internal constructor () {
         LibGleanFFI.INSTANCE.glean_destroy_glean(handle, e)
         handle = 0L
     }
+
+    /**
+     * Register a [PingType] in the registry associated with this [Glean] object.
+     */
+    internal fun registerPingType(pingType: PingType) {
+        if (handle == 0L) {
+            pingTypeQueue.add(pingType)
+        } else {
+            LibGleanFFI.INSTANCE.glean_register_ping_type(
+                handle,
+                pingType.handle
+            )
+        }
+    }
+
+    /**
+     * Returns true if a ping by this name is in the ping registry.
+     *
+     * For testing only.
+     */
+    // @VisibleForTesting(otherwise = VisibleForTesting.NONE)
```

Comment:
nit: let's remove the commented line if not needed.

Reply:
Oh, shoot.  It should be there, just committed without removing my playing around here.